src/pages/Profile/ProfileStat.css

```diff
@@ -0,0 +1,14 @@
+.profile-stat {
+    display: flex;
+    flex-direction: column;
+    justify-content: flex-start;
+}
+
+.profile-stat p {
+    margin: 0;
+}
+
+
+.profile-stat-num {
+    font-size: 4rem;
```

Comment:
it's good to have a `line-height` attribute alongside any `font-size` that isn't the pre-defined one so that your text will have room to space itself out; figma will usually have this in the text style properties too